idaes/power_generation/flowsheets/subcritical_power_plant/subcritical_boiler_flowsheet.py

```diff
@@ -0,0 +1,1311 @@
+    iscale.set_scaling_factor(fs.aPlaten.control_volume.energy_holdup, 1e-8)
+    iscale.set_scaling_factor(fs.aPlaten.control_volume.material_holdup, 1e-6)
+    iscale.set_scaling_factor(fs.aPipe.control_volume.energy_holdup, 1e-9)
+    iscale.set_scaling_factor(fs.aPipe.control_volume.material_holdup, 1e-5)
+    iscale.set_scaling_factor(fs.aBoiler.waterwall_heat, 1e-7)
+    iscale.set_scaling_factor(fs.aBoiler.platen_heat, 1e-7)
+    iscale.set_scaling_factor(fs.aBoiler.roof_heat, 1e-7)
+    iscale.set_scaling_factor(fs.aECON.shell._enthalpy_flow, 1e-8)
+    iscale.set_scaling_factor(fs.aECON.tube._enthalpy_flow, 1e-8)
+    iscale.set_scaling_factor(fs.aECON.shell.enthalpy_flow_dx, 1e-7)
+    iscale.set_scaling_factor(fs.aECON.tube.enthalpy_flow_dx, 1e-7)
+    for t, c in fs.aECON.shell.enthalpy_flow_dx_disc_eq.items():
+        iscale.constraint_scaling_transform(c, 1e-7)
+    for t, c in fs.aECON.tube.enthalpy_flow_dx_disc_eq.items():
+        iscale.constraint_scaling_transform(c, 1e-7)
+
+    iscale.set_scaling_factor(fs.aPSH.shell._enthalpy_flow, 1e-8)
+    iscale.set_scaling_factor(fs.aPSH.tube._enthalpy_flow, 1e-8)
+    iscale.set_scaling_factor(fs.aPSH.shell.enthalpy_flow_dx, 1e-7)
+    iscale.set_scaling_factor(fs.aPSH.tube.enthalpy_flow_dx, 1e-7)
+    for t, c in fs.aPSH.shell.enthalpy_flow_dx_disc_eq.items():
+        iscale.constraint_scaling_transform(c, 1e-7)
+    for t, c in fs.aPSH.tube.enthalpy_flow_dx_disc_eq.items():
+        iscale.constraint_scaling_transform(c, 1e-7)
+
+    iscale.set_scaling_factor(fs.aRH1.shell._enthalpy_flow, 1e-8)
+    iscale.set_scaling_factor(fs.aRH1.tube._enthalpy_flow, 1e-8)
+    iscale.set_scaling_factor(fs.aRH1.shell.enthalpy_flow_dx, 1e-7)
+    iscale.set_scaling_factor(fs.aRH1.tube.enthalpy_flow_dx, 1e-7)
+    for t, c in fs.aRH1.shell.enthalpy_flow_dx_disc_eq.items():
+        iscale.constraint_scaling_transform(c, 1e-7)
+    for t, c in fs.aRH1.tube.enthalpy_flow_dx_disc_eq.items():
+        iscale.constraint_scaling_transform(c, 1e-7)
+
+    iscale.set_scaling_factor(fs.aRH2.shell._enthalpy_flow, 1e-8)
+    iscale.set_scaling_factor(fs.aRH2.tube._enthalpy_flow, 1e-8)
+    iscale.set_scaling_factor(fs.aRH2.shell.enthalpy_flow_dx, 1e-7)
+    iscale.set_scaling_factor(fs.aRH2.tube.enthalpy_flow_dx, 1e-7)
+    for t, c in fs.aRH2.shell.enthalpy_flow_dx_disc_eq.items():
+        iscale.constraint_scaling_transform(c, 1e-7)
+    for t, c in fs.aRH2.tube.enthalpy_flow_dx_disc_eq.items():
+        iscale.constraint_scaling_transform(c, 1e-7)
+
+    # Calculate calculated scaling factors
+    iscale.calculate_scaling_factors(m)
+
+
+def main_steady_state():
+    m_ss = get_model(dynamic=False)
+    aBoiler = m_ss.fs_main.fs_blr.aBoiler
+    print('PA flow=',
+          pyo.value(aBoiler.primary_air[0].flow_mass))
+    print('SA flow=',
+          pyo.value(aBoiler.secondary_air[0].flow_mass))
+    print('O2 in flue gas', pyo.value(aBoiler.fluegas_o2_pct_dry[0]))
```

Comment:
Again maybe log if this isn't the main results or if they are super important results, maybe send to a text file.